src/main/kotlin/racingcar/Validation.kt

```diff
@@ -0,0 +1,22 @@
+package racingcar
+
+class Validation {
+
+    fun validateCarNames(carNames: List<String>): List<String> {
+        return carNames.apply {
+            forEach { validateCarName(it) }
+        }
+    }
```

Comment:
단순 유효성 검사이기 때문에 반환할 필요가 없어보여요! 
```suggestion
    fun validateCarNames(carNames: List<String>) {
        carNames.forEach { validateCarName(it) }
    }
```

Reply:
그렇네요..! 놓친 부분 이였는데 감사합니다 👍